app/scripts/views/filelist.js

```diff
@@ -1,34 +1,26 @@
-import React from 'react'
-import $ from 'jquery'
+import React, { Component } from 'react'
 import i18n from '../utils/i18n.js'
-import {Table, Tooltip, OverlayTrigger} from 'react-bootstrap'
+import {Table} from 'react-bootstrap'
+import FileItem from './fileitem'
+import Debug from 'debug'
```

Comment:
Agreed that's why I switched to using lower case and then const log = debug('mycomponent')